testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/jackson2/CustomJackson2ProviderTest.java

```diff
@@ -42,6 +45,12 @@
         WebArchive war = TestUtil.prepareArchive(CustomJackson2ProviderTest.class.getSimpleName());
         war.addClasses(CustomJackson2ProviderApplication.class, CustomJackson2ProviderResource.class);
         war.addAsWebInfResource(CustomJackson2ProviderTest.class.getPackage(), "jboss-deployment-structure-exclude-jaxrs.xml","jboss-deployment-structure.xml");
+        war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(
+                new RuntimePermission("getProtectionDomain"),
+                new ReflectPermission("suppressAccessChecks"),
+                new PropertyPermission("resteasy.allowGzip", "read"),
+                new PropertyPermission("resteasy.server.tracing.*", "read")
```

Comment:
for the "resteasy.xyz" properties, we could actually decide to use privileged actions in the code and avoid asking users to grant permissions. Maybe create an additional jira for this?

Reply:
ResteasyDeployment line 101, 102 is calling system property
      tracingText = System.getProperty(ResteasyContextParameters.RESTEASY_TRACING_TYPE);
      thresholdText = System.getProperty(ResteasyContextParameters.RESTEASY_TRACING_THRESHOLD);

wfly security manager does not allow code to override the call to System.getProperty.
This the same issue we have with PortProviderUtil.

Reply:
OK